doc/roadmap.md

```diff
@@ -1,92 +1,59 @@
 # Roadmap
 
-This page includes an ambitious roadmap for hvPlot for 2024.
+## hvPlot 1.0
 
-## 1. Documentation, documentation, documentation
+### 1. Documentation
 
-### Context
+In 2025, we have obtained a Small Development Grant from NumFocus that allowed us to significantly update the documentation, adding a new Reference section with a description of all the plotting options and an automated API reference, a new examples gallery, and new tutorials.
 
-hvPlot's website currently has the following structure:
+To complete this documentation update, we need now to convert the user guides into How-To guides, Tutorials, and Explanation pages.
 
-- A landing page describing its main features.
-- A *Getting Started* guide (recently added) that is a short tutorial demonstrating its basic capabilities.
-- Many *User Guides*, focusing on a domain (e.g. *Geographic Data*), on a data type (*Gridded Data*), on a feature (e.g. *Interactive*), or on a specific part of the API (e.g. *Customization*).
-- A *Reference Gallery* with a page per plot method (actually incomplete), separated by these categories: Pandas, GeoPandas, and Xarray. Each reference page is usually short, only uses Bokeh, and doesn't include the whole API applicable to the plot method.
-- A *Topics* short page containing a few links to other websites using hvPlot
-- A *Developer Guide*, a *Releases* page and an *About* page
+### 2. Refactor the test suite
 
-hvPlot users have repeatedly complained about the quality of its documentation, which seems to mostly stem from:
+The test suite should be re-factored and improved to catch bugs before users have to report them and to prevent potential regressions.
```

Comment:
Would you like to add testing for cuDF, even if only to run locally?

Reply:
That's something I'm gonna do but would be happy if you want to do it. I don't think there's a need to add it to the roadmap though.